stacks/dovetail.prx.org.yml

```diff
@@ -387,6 +442,9 @@ Resources:
             !Sub ${InfrastructureStorageStackName}-InfrastructureSupportBucket
         S3Key: utility/ecs-service-size-rollover.zip
       Description: Allows for DesiredCount persistence on ECS services
+      Environment:
+        Variables:
+          DEFAULT_DESIRED_COUNT: 10
```

Comment:
May want a lower default, for now.  To avoid having to spin up 10 additional staging instances to deploy this.  Maybe "2"?